src/fontra/views/applicationsettings/panel-shortcuts.js

```diff
@@ -0,0 +1,496 @@
+            type: "auxiliaryElement",
+            key: ["label", key],
+            auxiliaryElement: html.createDomElement("div", {
+              "class": "ui-form-label universal-row",
+              "style": `cursor: pointer;`,
+              "innerHTML": translate(key, ""),
+              "data-tooltip": "click for editing",
+              "data-tooltipposition": "top",
+              "onclick": (event) => this.doEditShortCut(key),
+            }),
+          },
+          field2: {
+            type: "edit-text-shortcut",
+            key: [key],
+            value: buildShortCutString(shortCutDefinition),
+            allowEmptyField: true,
+            style: `width: 6em; text-align: center;`,
+          },
+          field3: {
+            "type": "checkbox",
+            "key": ["globalOverride", key],
+            "value": shortCutDefinition
+              ? shortCutDefinition.globalOverride || false
+              : false,
+            "data-tooltip": "Global Override",
+            "data-tooltipposition": "top",
+          },
+        });
+      }
+
+      this.infoForm.setFieldDescriptions(formContents);
+      this.infoForm.onFieldChange = async (fieldItem, value, valueStream) => {
+        const isGlobalButton = fieldItem.key[0] === "globalOverride";
+        const shortCutKey = isGlobalButton ? fieldItem.key[1] : fieldItem.key[0];
+
+        const globalOverrideDef = isGlobalButton
+          ? value
+          : shortCutsData[shortCutKey] === null
+          ? false
+          : shortCutsData[shortCutKey].globalOverride || false;
+        const shortCutDefinition =
+          isGlobalButton && shortCutsData[shortCutKey] === null
+            ? null
+            : isGlobalButton
+            ? { ...shortCutsData[shortCutKey], globalOverride: globalOverrideDef }
+            : parseShortCutString(value, globalOverrideDef);
+
+        const warnings = validateShortCutDefinition(shortCutKey, shortCutDefinition);
+        if (warnings.length > 0) {
+          message(
+            `Invalid ShortCut "${buildShortCutString(
+              shortCutDefinition
+            )}" for "${translate(shortCutKey, "")}":`,
+            warnings.join("\n")
+          );
+          this.setupUI();
+          return;
+        }
+        this.saveShortCut(shortCutKey, shortCutDefinition);
+      };
+
+      this.panelElement.appendChild(this.infoForm);
+    }
+  }
+
+  async doEditShortCut(key) {
+    const shortCutDefinition = await doEditShortCutDialog(key);
+    const newShortCutDefinition = _shortCutDefinitionNormalized(shortCutDefinition);
+    if (newShortCutDefinition === undefined) {
+      // User cancelled, do nothing.
+      return;
+    }
+    this.saveShortCut(key, newShortCutDefinition);
+  }
+
+  saveShortCut(key, newShortCutDefinition) {
+    shortCutsData[key] = newShortCutDefinition;
+    shortCutsDataCustom[key] = newShortCutDefinition;
+
+    localStorage.setItem("shortCuts-custom", JSON.stringify(shortCutsDataCustom));
```

Comment:
@justvanrossum And this is wehre it is saved to localStorage. But `saveShortCut` will be moved, once I am done with the rework of the UI so the it does not use ui-form anymore.